algo_marg_settings.asciipb

```diff
@@ -25,3 +25,12 @@ init_num_clusters: 3
 # (NOTE: 3 is the recommended value in most cases, please change it only if you
 #  know what you're doing.)
 neal8_n_aux: 3
+
+# Split and Merge number of restricted GS scans for each MH step.
+splitmerge_n_restr_gs_updates: 5
+
+# Split and Merge number of MH updates for each iteration
+splitmerge_n_mh_updates: 1
+
+# Split and Merge number of full GS scans for each iteration
+splitmerge_n_full_gs_updates: 1
```

Comment:
This missing EOF newline, as well as other stuff in this PR, tells me you have not installed the pre-commit hook. Please follow instructions in [CONTRIBUTORS.md](https://github.com/bayesmix-dev/bayesmix/blob/master/CONTRIBUTORS.md) so that stuff like this is automatically taken care of whenever you create a new commit (you may need to re-add modified files to the commit before pushing). Actually I'm not sure if the hook only takes care of C++ files. Therefore, please check that this particular newline is there before pushing, and fix it manually if needed

Reply:
We installed the pre-commit hooks starting from commit [fec8db5](https://github.com/bayesmix-dev/bayesmix/pull/116/commits/fec8db54494b747eed5695ea43d04df82fa484b3), indeed the code in C++ is formatted differently w.r.t. how we wrote it. 

We will fix the newline. Are there other formatting errors that we can fix manually?

Reply:
I could have sworn I had seen a line with over 79 characters, but now it's nowhere to be found. My bad

Reply:
"don't drink and revise"!